src/cmdstan-guide/generate_quantities_config.qmd

```diff
@@ -1,21 +1,75 @@
 ---
-pagetitle: Standalone Generate Quantities
+pagetitle: Generating Quantities of Interest from a Fitted Model
 ---
 
-# Standalone Generate Quantities
+# Generating Quantities of Interest from a Fitted Model  {#gc-intro}
 
 The `generate_quantities` method allows you to generate additional
 quantities of interest from a fitted model without re-running the sampler.
-For an overview of the uses of this feature, see the
-[QuickStart Guide section](generate_quantities_intro.qmd)
-and the Stan User's Guide section on
-[Stand-alone generated quantities and ongoing prediction](https://mc-stan.org/docs/stan-users-guide/posterior-prediction.html#stand-alone-generated-quantities-and-ongoing-prediction).
+Instead, you write a modified version of the original Stan program
+and add a generated quantities block or modify the existing one
+which specifies how to compute the new quantities of interest.
+Running the `generate_quantities` method on the new program
+together with sampler outputs (i.e., a set of draws)
+from the fitted model runs the generated quantities block
+of the new program using the the existing sample by plugging
+in the per-draw parameter estimates for the computations in
+the generated quantities block.
 
 This method requires sub-argument `fitted_params` which takes as its value
-an existing Stan CSV file that contains a sample from an equivalent model,
-i.e., a model with the same parameters, transformed parameters, and model blocks,
+an existing [Stan CSV](stan_csv_apdx.qmd) file that contains a parameter values
+from an equivalent model, i.e., a model with the same parameters and transformed parameters blocks,
```

Comment:
Is there any reason we need to keep the transformed parameters the same?  We should be able to rerun with just the same parameters.